Deferred/ReadWriteLock.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2014-2015 Big Nerd Ranch. Licensed under MIT.
 //
 
-import Foundation
+import Dispatch
```

Comment:
I don't see this file being used by anything other than tests.  If it isn't needed, I think it should be removed from the repo.